datalad/tests/utils.py

```diff
@@ -527,7 +527,8 @@ def newfunc(*args, **kwargs):
 
 @with_tempfile(mkdir=True)
 def get_most_obscure_supported_name(tdir):
-    """Return the most filename which filesystem under TEMPDIR could support
+    """Return the most obscure filename which the filesystem under TEMPDIR 
+       could support
```

Comment:
ideally the leading summary line should be a single line (and then empty line, and then detailed description).  so just remove (`under TEMPDIR`) I guess and embed that information into detailed description